config/lava/chromeos/tast.jinja2

```diff
@@ -0,0 +1,30 @@
+- test:
+      timeout:
+        minutes: 45
+      docker:
+        image: {{ docker_image }}
+        wait:
+          device: true
+      results:
+          location: /home/cros-tast/lava
+      definitions:
+      - repository:
+          metadata:
+            format: Lava-Test Test Definition 1.0
+            name: docker-ping
+          run:
+            steps:
+              - 'cd /home/cros-tast'
+{%- if cros_tast_tarball %}
+              - 'curl -s {{cros_tast_tarball}} | tar xjf -'
+{% endif %}
+{%- if rootfs_url %}
+              - 'curl -s {{rootfs_url}}tast.tgz | tar xzf -'
+              - 'cp remote_test_runner /usr/bin/remote_test_runner'
+{% endif %}
+              - 'while ! ping -c 1 -w 1 $(lava-target-ip); do sleep 1; done'
+              - 'ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i /home/cros-tast/.ssh/id_rsa root@$(lava-target-ip) whoami'
+              - './tast-parser.py example.ARCFixture example.ManyParams.arc_state example.Param.dog example.Pass example.Fail'
+        from: inline
+        name: docker-tast-tests
+        path: inline/docker-ping.yaml
```

Comment:
Ditto

Reply:
Fixed and tested in https://lava.collabora.co.uk/scheduler/job/5987415